docs/usage/io.rst

```diff
@@ -0,0 +1,36 @@
+Prediction Using IO Operations
+==============================
+
+This example will show you how to train a model using DFFML python API and use the
+model for prediction by taking input from stdio.
+
+DFFML offers several :ref:`plugin_models`. For this example
+we will be using the Simple Linear Regression model
+( ``slr`` ) which is in the ``dffml`` package.
+
```

Comment:
@pdxjohnny Putting link to `Models` page. 
:ref:`plugin_model_dffml_slr` doesn't seem to work. I tried putting an anchor for `slr` to reference it but am unable to do it because `dffml_model.rst` gets overwritten. What am I missing?

Reply:
Have you merged master? I just added the references recently

Reply:
Ohh! Thanks! Done.